src/js/providers/html5.js

```diff
@@ -443,8 +446,11 @@ define([
                 return;
             }
 
-            _setLevels(item.sources);
-            this.sendMediaType(item.sources);
+            // check if _setLevels has already been called
```

Comment:
Find the place where the event is exiting into the public API - ie through
the on('levels') and filter it out there

On Mon, Sep 21, 2015 at 1:59 PM, pajong <notifications@github.com> wrote:

> In src/js/providers/html5.js
> <https://github.com/jwplayer/jwplayer/pull/727#discussion_r40003279>:
>
> > @@ -443,8 +446,11 @@ define([
> >                  return;
> >              }
> >
> > -            _setLevels(item.sources);
> > -            this.sendMediaType(item.sources);
> > +            // check if _setLevels has already been called
>
> What do you mean by suppressing the event from the public API?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jwplayer/jwplayer/pull/727/files#r40003279>.
>